Makefile

```diff
@@ -35,7 +35,8 @@ endif
 .PHONY: check-test
 
 load-test: check-test setup-venv
-	.venv/bin/locust -f $(TEST)_locustfile.py
+	./bin/start_locust.sh $(TEST)_locustfile.py
```

Comment:
Having such a script is excellent. I wonder if we should just give it the `$(TEST)` as argument, and then let it figure things out internally. That would get us closer to having a "CLI" for this.

Could also give this a shebang and drop the `.sh` extension.

Reply:
Ok, maybe later